### DISCUSSION TITLE
Sections: Online, Everything, Waiting, Blocked, Add to friends

### DISCUSSION THREAD

**Discussion by Green-Snake345** [stoatchat/discussions#712] [Feature Suggestions]
You can make sections such as "Online", "All", "Waiting", "Blocked", "Add to friends", do it in Revolt. If you like my idea, then put a reaction 👍

**Comment by Rexogamer** [comment]
Can you clarify what you mean here?

**Reply by Green-Snake345** [reply]
I wanted to say that you need to add sections such as: "Online", "All", "Waiting", "Blocked". In the "Online" section, a list of users who are online will be displayed, even if they have the status "Inactive", "Do not disturb", "Streaming", do the same as in discord
